src/main/java/adam/TaskList.java

```diff
@@ -0,0 +1,167 @@
+package adam;
+
+import java.util.ArrayList;
+
+import adam.tasks.Deadline;
+import adam.tasks.Event;
+import adam.tasks.Task;
+import adam.exception.OutOfBoundException;
+import adam.tasks.ToDo;
+
+/**
+ * This class holds all the methods that is realted to the task list such as adding and deleting tasks.
+ */
+public class TaskList {
+    private Ui ui = new Ui();
```

Comment:
Not sure if a TaskList should be constructing the UI again if it was constructed once in Adam.java